arviz/plots/backends/bokeh/autocorrplot.py

```diff
@@ -59,10 +60,8 @@ def plot_autocorr(
         ax.title = title
 
     if axes.size > 0:
-        axes[0, 0].x_range._property_values["start"] = 0  # pylint: disable=protected-access
-        axes[0, 0].x_range._property_values["end"] = max_lag  # pylint: disable=protected-access
-        axes[0, 0].y_range._property_values["start"] = -1  # pylint: disable=protected-access
-        axes[0, 0].y_range._property_values["end"] = 1  # pylint: disable=protected-access
+        axes[0, 0].x_range = DataRange1d(bounds="auto")  # pylint: disable=protected-access
+        axes[0, 0].y_range = DataRange1d(bounds="auto")  # pylint: disable=protected-access
```

Comment:
add `start` and `end` values to `DataRange1d` function.

Reply:
@ahartikainen, setting ```start``` and ```end``` values of ```DataRange1d``` for ```x_range``` to 0 and ```max_lag``` and for ```y_range``` to -1 and 1 and then setting bounds to ```auto``` restricts the plot to these values only and the entire plot cannot be panned.

So, in order to keep the given ranges to viewport while limiting pan values to the entire range I need to set the value of ```bounds``` to the entire range. How do I get this value of entire ```x_range``` and ```y_range```?

Reply:
for x_range values are from 0 --> `np.arange(len(y))`
fir y_range values are from -1 --> 1

Reply:
Thanks for the help! Also, adding ```min_interval``` to ```DataRange1d``` limits the zoom in value, I've set it to ```5``` (float) for ```plot_autocorr```. Can you suggest the values for other plots?